src/test/resources/com/puppycrawl/tools/checkstyle/javadocpropertiesgenerator/InputJavadocPropertiesGeneratorCorrect2.java

```diff
@@ -0,0 +1,16 @@
+package com.puppycrawl.tools.checkstyle.javadocpropertiesgenerator;
+
+public class InputJavadocPropertiesGeneratorCorrect2 {
+
+    // note: &#064; is the html escape for '@',
+    // used here to avoid confusing the javadoc tool
+    /**
+     * An annotation of a package, type, field, parameter or variable.
+     * An annotation may occur anywhere modifiers occur (it is a
+     * type of modifier) and may also occur prior to a package definition.
+     * The notable children are: The annotation name and either a single
+     * default annotation value or a sequence of name value pairs.
+     * Annotation values may also be annotations themselves.
+     */
+    public static final @Deprecated int EOF1 = 1, EOF2 = 2;
```

Comment:
We never Going to do this if our token Java class.
Having same javadoc for different token is nonsense.
Technically possible, but nonsense.
Why we need test for nonsense ?
This generator class is internal tool, it should work only on out TokenTypes.java.

 Is there a way to refactor java code to avoid mutation survival and not supporting of this type of javadoc on multiple?
I would rather have let java code to throw exception in such code to clearly highlight it not good way to write a code in token class.

Reply:
I have make it simple.
the difference is becuase of the single line comment above the javadoc.